tests/nl2sql/test_enhanced_summary.py

```diff
@@ -0,0 +1,25 @@
+import ast
+from pathlib import Path
+
+
+def test_enhanced_summary():
+    from nl2sql.enhanced_summary import enhanced_summary
+    from tests.utils.summary_category_extractor import \
+        summary_category_extractor
+
+    summary = enhanced_summary(
+        docs=Path("docs/martin_brundle.txt").read_text(),
+        user_query="Tell me about Martin Brundle",
+        sql_results=[('Martin', 'Brundle', '1959-06-01', 'British')],
+    )
+
+    categories = ast.literal_eval(summary_category_extractor(
```

Comment:
consider "hiding" the dictionary evaluation inside the test utility, so it is easier to use.